### PR TITLE
chore: address remarks

### PR DESCRIPTION
**Pull request by ntn-x2**
Fixes https://github.com/KILTprotocol/ticket/issues/2412.

Addresses open remarks, since a couple were already closed as part of other PRs.

### CODE REVIEW

**Comment by kilted-andres** [comment]
Looks good to me. 😄 👍

File: docs/develop/03_workshop/04_attester/01_account.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ A person can memorize it, and use it later to re-generate their keypairs and add
 To generate an account, one method from the Polkadot crypto utility, and one method from the KILT SDK is needed:
 
 - `mnemonicGenerate()` - Generates a 12-word mnemonic
-- `makeKeypairFromSeed(seed)` - takes a mnemonic as an input, and outputs a `Keypair` instance.
+- `makeKeypairFromSeed(seed)` - takes a mnemonic as an input, and outputs a [`KiltKeyringPair`](https://kiltprotocol.github.io/sdk-js/interfaces/_kiltprotocol_types.KiltKeyringPair.html) instance. A `Keypair` and a `KeyringKeypair` different in what information they expose. While a `Keypair` exposes both its public and private keys, a `KeyringPair` is managed by a keyring, which does not allow exposing private keys, but only supports signing arbitrary payloads.
```
Comment (rompemoldes) [inline docs/develop/03_workshop/04_attester/01_account.md]: A `Keypair` and a `KeyringKeypair` **differ** in what information they expose.

**Review by rompemoldes — CHANGES_REQUESTED** [review]
Looks good to me. 😄 👍 

Only a small grammar error.

File: docs/develop/03_workshop/04_attester/01_account.md (hunk 1 above)
Comment (Ad96el) [inline docs/develop/03_workshop/04_attester/01_account.md]: `makeKeyParFromSeed` is not used in the code snippet below. There is another approach shown with `keyring.addFromMnemonic(mnemonic)` 

image

Comment (ntn-x2) [reply docs/develop/03_workshop/04_attester/01_account.md]: Good point: https://github.com/KILTprotocol/docs/pull/233/commits/63321d3d96f679eec9bb7e55900bd1118303f0bf

Comment (ntn-x2) [reply docs/develop/03_workshop/04_attester/01_account.md]: I fixed this piece based on Adel's feedback: https://github.com/KILTprotocol/docs/pull/233/commits/63321d3d96f679eec9bb7e55900bd1118303f0bf.

File: docusaurus.config.js [hunk 2]
```diff
@@ -211,6 +211,12 @@ module.exports = {
         theme: {
           customCss: require.resolve('./src/css/custom.css'),
         },
+        // sitemap: {
```
Comment (Ad96el) [inline docusaurus.config.js]: Why is this commented out?

Comment (ntn-x2) [reply docusaurus.config.js]: Ops... Cross-branch contamination 😆 It was not supposed to be there in the first place. Fixed that https://github.com/KILTprotocol/docs/pull/233/commits/4f805f411812d9347a4ccdc4a85633cd43db0a28.

**Review by Ad96el — APPROVED** [review]
LGTM!

**Pull request merged**